packages/preview-middleware/src/types/index.ts

```diff
@@ -53,6 +53,11 @@ export interface FlpConfig {
     libs?: boolean;
     apps: App[];
     theme?: string;
+    test?: {
+        framework?: string;
+    };
+    fioriSandboxConfig?: boolean;
```

Comment:
It is not part of the corresponding feature request. Additionally, I'd challenge that it is necessary since you can define the same with the exist `apps` part of the middleware config.
using a custom FLP sandbox config is also bound to the existing FLP sandbox implementation, it might not be there in new versions anymore.

Please remove the `fioriSandboxConfig` property. We can have a separate discussion whether to add something like it in the future.

Reply:
Fine for me 👍🏻 will remove it and create a separate PR for it (and the discussions 😸).